src/services_models/reserves.jl

```diff
@@ -31,7 +33,15 @@ function service_requirement_constraint!(
     @debug initial_time
     time_steps = get_time_steps(container)
     name = PSY.get_name(service)
-    constraint = get_constraint(container, RequirementConstraint(), SR)
+    add_cons_container!(
+        container,
+        RequirementConstraint(),
+        SR,
+        [name],
+        time_steps;
+        meta = name,
+    )
+    constraint = get_constraint(container, RequirementConstraint(), SR, name)
```

Comment:
you don't need to retrieve the constraint after creation you can just do `constraint = add_cons_container(...)`